### PR TITLE
@streams doc

### PR DESCRIPTION
**Pull request by tbeseda**
Add doc for `@tables-streams` pragma and update related docs.

I'll need a review or two double checking the technical details since this feature hasn't been documented before.
Thanks!

### CODE REVIEW

File: src/views/docs/en/reference/project-manifest/indexes.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ description: Define DynamoDB table global secondary indexes.
 
 Defines [Global Secondary Indexes][gsi] for your project's [DynamoDB][ddb] tables. `@indexes` should only ever be paired with [`@tables`][tables].
 
+> ℹ️  As of Architect v9.4, `@tables-indexes` can be used in place of `@indexes`. `@indexes` will be superseded in a future Arc release.
```
Comment (ryanblock) [inline src/views/docs/en/reference/project-manifest/indexes.md:9]: `can be used` → `should be used`.

File: src/views/docs/en/reference/project-manifest/tables-streams.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+title: '@tables-streams'
+category: app.arc
+description: Define DynamoDB tables with streaming changes
+---
+
+Define Lambda functions for streaming changes to DynamoDB tables. Respond to `insert`, `update`, and `destroy` events with a handler function.
+
+> ⚠️  Unfortunately, "dynalite" (used under the hood in Architect [Sandbox](../cli/sandbox)) doesn't support streams, so emulation isn't yet supported for local development.
+
+## Recommended Resources
```
Comment (ryanblock) [inline src/views/docs/en/reference/project-manifest/tables-streams.md:11]: Personally I prefer to put these kinds of things at the bottom unless they are required reading.

File: src/views/docs/en/reference/project-manifest/tables-streams.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+[AWS DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.Lambda.html)
+
+## Syntax
+
+- Name
+  - Lowercase alphanumeric string
+  - Must match a `@tables` name
+
+Table streams can use more verbose configuration to allow for [custom source paths](../../guides/developer-experience/custom-source-paths) and names in your project. Optionally provide a `name` and/or `src` for each table stream.
```
Comment (ryanblock) [inline src/views/docs/en/reference/project-manifest/tables-streams.md:21]: Might be worth mentioning this is how you can specify multiple distinct table stream lambdas for a single table.

File: src/views/docs/en/reference/project-manifest/tables-streams.md (hunk 2 above)
Comment (tbeseda) [reply src/views/docs/en/reference/project-manifest/tables-streams.md:11]: Agreed. it kind of interrupts the doc too much

**Pull request merged**